### PR TITLE
[Improve] Localization streampark-usercase-shunwang

### PR DESCRIPTION
**Pull request by VampireAchao**
see: https://github.com/apache/incubator-streampark/issues/3218

cc: @caicancai @saLeox

### CODE REVIEW

File: blog/4-streampark-usercase-shunwang.md [hunk 1]
```diff
@@ -1,213 +1,213 @@
...
-- 遇到的挑战
-- 为什么用 StreamPark
-- 落地实践
-- 带来的收益
-- 未来规划
+- Introduction to the company's business
+- Challenges encountered
+- Why choose StreamPark
+- Implementation in practice
+- Benefits realized
```
Comment (saLeox) [inline blog/4-streampark-usercase-shunwang.md:16]: Benefits Brought

File: blog/4-streampark-usercase-shunwang.md [hunk 2]
```diff
@@ -1,213 +1,213 @@
...
-面对自研平台 Streaming-Launcher 存在的缺陷，我们一直在思考如何将 Flink 的使用门槛降到更低，进一步提高工作效率。考虑到人员投入成本和时间成本，我们决定向开源社区求助寻找合适的开源项目来对我们的 Flink 任务进行管理和运维。
+Faced with the defects of our self-developed platform Streaming-Launcher, we have been considering how to further lower the barriers to using Flink and improve work efficiency. Considering the cost of human resources and time, we decided to seek help from the open-source community and look for an appropriate open-source project to manage and maintain our Flink tasks.
 
 
 
-### 01  **StreamPark 解决 Flink 问题的利器**
+### 01  **StreamPark: A Powerful Tool for Solving Flink Issues**
 
-很幸运在 2022 年 6 月初，我们在 GitHub 机缘巧合之间认识到了 StreamPark，我们满怀希望地对 StreamPark 进行了初步的探索。发现 StreamPark 具备的能力大概分为三大块：用户权限管理、作业运维管理和开发脚手架。
+Fortunately, in early June 2022, we stumbled upon StreamPark on GitHub and embarked on a preliminary exploration full of hope. We found that StreamPark's capabilities can be broadly divided into three areas: user rights management, job operation and maintenance management, and development scaffolding.
```
Comment (saLeox) [inline blog/4-streampark-usercase-shunwang.md:82]: user permission management

File: blog/4-streampark-usercase-shunwang.md [hunk 3]
```diff
@@ -1,213 +1,213 @@
...
 
-**用户权限管理**
+**User Rights Management**
```
Comment (saLeox) [inline blog/4-streampark-usercase-shunwang.md:84]: **User Permission Management**

File: blog/4-streampark-usercase-shunwang.md [hunk 4]
```diff
@@ -1,213 +1,213 @@
...
 
-在 StreamPark 平台中为了避免用户权限过大，发生一些不必要的误操作，影响作业运行稳定性和环境配置的准确性，提供了相应的一些用户权限管理功能，这对企业级用户来说，非常有必要。
+In the StreamPark platform, to avoid the risk of users having too much authority and making unnecessary misoperations that affect job stability and the accuracy of environmental configurations, some user rights management functions are provided. This is very necessary for enterprise-level users.
```
Comment (saLeox) [inline blog/4-streampark-usercase-shunwang.md:86]: user rights management -> user permission management

File: blog/4-streampark-usercase-shunwang.md [hunk 5]
```diff
@@ -1,213 +1,213 @@
...
+At Shunwang Technology, we have developed the ODPS platform based on the Flink-SQL-Gateway to facilitate business developers in managing the metadata of Flink tables. Business developers perform DDL operations on Flink tables in ODPS, and then carry out analysis and query operations on the created Flink tables in StreamPark. Throughout the entire business development process, we have decoupled the creation and analysis of Flink tables, making the development process appear more straightforward.
 
-开发同学如果想在 ODPS 上查询实时数据，我们需要提供一个 Flink SQL 的运行环境。我们使用 StreamPark 运行了一个 Yarn Session 的 Flink 环境提供给 ODPS 做实时查询。
+If developers wish to query real-time data in ODPS, we need to provide a Flink SQL runtime environment. We have used StreamPark to run a Yarn Session Flink environment to support ODPS in conducting real-time queries.
 
-![图片](/blog/shunwang/homework.png)
+![Image](/blog/shunwang/homework.png)
 
-目前 StreamPark 社区为了进一步降低实时作业开发门槛，正在对接 Flink-SQL-Gateway。
+Currently, the StreamPark community is interfacing with Flink-SQL-Gateway to further lower the barriers to developing real-time jobs.
```
Comment (saLeox) [inline blog/4-streampark-usercase-shunwang.md:156]: the StreamPark community is intergrating with Flink-SQL-Gateway

File: blog/4-streampark-usercase-shunwang.md [hunk 6]
```diff
@@ -1,213 +1,213 @@
...
+![Image](/blog/shunwang/achievements1.png)
 
-![图片](/blog/shunwang/achievements2.png)
+![Image](/blog/shunwang/achievements2.png)
 
-##  未 来 规 划
+## Future Plans
 
-顺网科技作为 StreamPark 早期的用户之一，在 1 年期间内一直和社区同学保持交流，参与 StreamPark 的稳定性打磨，我们将生产运维中遇到的 Bug 和新的 Feature 提交给了社区。在未来，我们希望可以在 StreamPark 上管理 Flink 表的元数据信息，基于 Flink 引擎通过多 Catalog 实现跨数据源查询分析功能。目前 StreamPark 正在对接 Flink-SQL-Gateway 能力，这一块在未来对于表元数据的管理和跨数据源查询功能会提供了很大的帮助。
+As one of the early users of StreamPark, Shunwang Technology has been communicating with the community for a year and participating in the polishing of StreamPark's stability. We have submitted the Bugs encountered in production operations and new Features to the community. In the future, we hope to manage the metadata information of Flink tables on StreamPark, and implement cross-data-source query analysis functions based on the Flink engine through multiple Catalogs. Currently, StreamPark is interfacing with Flink-SQL-Gateway capabilities, which will greatly help in the management of table metadata and cross-data-source query functions in the future.
```
Comment (saLeox) [inline blog/4-streampark-usercase-shunwang.md:206]: StreamPark is integrating with Flink-SQL-Gateway

File: blog/4-streampark-usercase-shunwang.md [hunk 7]
```diff
@@ -1,213 +1,213 @@
...
 
-由于在顺网科技多是已 Yarn Session 模式运行的作业，我们希望 StreamPark 可以提供更多对于 Remote集群、Yarn Session 集群和 K8s Session 集群功能支持，比如监控告警，优化操作流程等方面。
+Since Shunwang Technology primarily runs jobs in Yarn Session mode, we hope that StreamPark can provide more support for Remote clusters, Yarn Session clusters, and K8s Session clusters, such as monitoring and alerts, and optimizing operational processes.
 
-考虑到未来，随着业务发展可能会使用 StreamPark 管理更多的 Flink 实时作业，单节点模式下的 StreamPark 可能并不安全。所以我们对于 StreamPark 的 HA 也是非常期待。
+Considering the future, as the business develops, we may use StreamPark to manage more Flink real-time jobs, and StreamPark in single-node mode may not be safe. Therefore, we are also looking forward to the High Availability (HA) of StreamPark.
 
-对于 StreamPark 对接 Flink-SQL-Gateway 能力、丰富 Flink Cluster 功能和 StreamPark HA，我们后续也会参与建设中。
+We will also participate in the construction of the capabilities of StreamPark interfacing with Flink-SQL-Gateway, enriching Flink Cluster functionality, and StreamPark HA.
```
Comment (saLeox) [inline blog/4-streampark-usercase-shunwang.md:212]: integrating with Flink-SQL-Gateway

**Comment by saLeox** [comment]
LGTM

**Review by caicancai — APPROVED** [review]

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**